src/core/public/chrome/chrome_service.tsx

```diff
@@ -245,7 +281,6 @@ export class ChromeService {
           badge$={badge$.pipe(takeUntil(this.stop$))}
           basePath={http.basePath}
           breadcrumbs$={breadcrumbs$.pipe(takeUntil(this.stop$))}
-          customNavLink$={customNavLink$.pipe(takeUntil(this.stop$))}
```

Comment:
Is it necessary to remove these props for now? I think it would better that we keep these props.

Reply:
Added